<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to AWAH SIP Codec</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Connect to Adress:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_State">
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_Port">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushButton_connect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>65353</number>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_connstate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
